src/core/issues-notes.js

```diff
@@ -203,6 +193,36 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (issueSummaryElement) {
```

Comment:
Avoid a level of nesting: 
```suggestion
  if (!issueSummaryElement) return;
```